vm/src/vm/runners/cairo_runner.rs

```diff
@@ -168,9 +172,12 @@ pub enum RunnerMode {
 }
 
 impl CairoRunner {
+    /// The `cairo_layout_params` argument should only be used with dynamic layout.
+    /// It is ignored otherwise.
     pub fn new_v2(
         program: &Program,
         layout: LayoutName,
+        cairo_layout_params: Option<CairoLayoutParams>,
```

Comment:
Can you change the name here to dynamic_layout_params, I think it is clearer

Reply:
Done